test/unit/spec/common/cellComponents/tabs/jobStatusTable-Spec.js

```diff
@@ -29,6 +29,11 @@ define([
     const allJobsWithBatchParent = JobsData.allJobsWithBatchParent;
     const allJobsNoBatchParent = JobsData.allJobs;
     const batchId = JobsData.batchParentJob.job_id;
+
```

Comment:
The changes in this file reflect moving over to using batch IDs for addressing job messages, and having all message type listeners added prior to table instantiation -- as would happen in the real bulk import cell.